.travis.yml

```diff
@@ -11,7 +11,7 @@ before_script:
   - mysql -e "create database IF NOT EXISTS JEngineV2;" -uroot
 
 script:
-  - mysql JEngineV2 < src/main/resources/JEngineV2_schema.sql -uroot
+  - mysql JEngineV2 < src/main/resources/chimera_db_schema.sql -uroot
```

Comment:
Since EclipseLink was introduced in Chimera 1.3 the sql script is no longer used to create the database schema. Instead, EclipseLink automatically creates the schema based on the @Enity annotations. `src/main/resources/chimer_db_schema.sql` should therefore be removed.